EDEngineer/Resources/Data/localization.json

```diff
@@ -1726,193 +1726,193 @@
       "de": null,
       "fr": null,
       "ru": null
     },
     "Signal source (anarchy)": {
-      "es": null,
+      "es": "Señales (anarquía)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Signal source (low security)": {
-      "es": null,
+      "es": "Señales (seguridad baja)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Signal source (High grade emissions)": {
-      "es": null,
+      "es": "Señales (alta calidad)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Ship salvage (military & authority ships)": {
-      "es": null,
+      "es": "Rescate de naves (naves militares y de la autoridad)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Destroyed Unknown Artefact": {
-      "es": null,
+      "es": "Destrucción de Artefacto Desconocido",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Ship scanning (combat ships)": {
-      "es": null,
+      "es": "Escaneo de naves (naves de combate)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Deep space data beacon": {
-      "es": null,
+      "es": "Baliza de datos en espacio profundo",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Ship scanning (transport ships)": {
-      "es": null,
+      "es": "Escaneo de naves (naves de transporte)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "High wake scanning": {
-      "es": null,
+      "es": "Escaneo de estelas de salto",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Surface data point": {
-      "es": null,
+      "es": "Punto de datos de superficie",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Ship scanning": {
-      "es": null,
+      "es": "Escaneo de naves",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Markets": {
-      "es": null,
+      "es": "Mercados",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Markets near Akhenaten (High Tech/Refinery)": {
-      "es": null,
+      "es": "Mercados cerca de Akhenaten (Alta Tecnología/Refinería)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Markets near Stafkarl (Industrial/Refinery)": {
-      "es": null,
+      "es": "Mercados cerca de Stafkarl (Industrial/Refinería)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Markets near Run (Industrial/Refinery)": {
-      "es": null,
+      "es": "Mercados cerca de Run (Industrial/Refinería)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Markets near Lei Jing (Industrial/Refinery)": {
-      "es": null,
+      "es": "Mercados cerca de Lei Jing (Industrial/Refinería)",
       "de": null,
       "fr": null,
       "ru": null
     },
     "Markets near Myrbat (Industrial/Refinery)": {
-      "es": null,
+      "es": Mercados cerca de Myrbat (Industrial/Refinería)",

```

Comment:
you can validate the entire json here : http://jsonlint.com/

(copy and paste the content of the file)